### PR TITLE
feat:implement hero section

### PR DESCRIPTION
**Pull request by LeVHoangduc**
@lenguyen-agilityio @hungnguyenhuu  Please help me to check PR and merge it.

### CODE REVIEW

File: practice-two/src/assets/css/pages/home/hero.css [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+.hero__image {
+  max-width: 100%;
+}
+
+.hero__banner {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
```
Comment (hungnguyenhuu) [inline practice-two/src/assets/css/pages/home/hero.css:35]: break-line?

File: practice-two/src/assets/css/pages/home/hero.css [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+  width: 100%;
+  height: 180px;
+
```
Comment (hungnguyenhuu) [inline practice-two/src/assets/css/pages/home/hero.css:38]: break-line?

Comment (LeVHoangduc) [reply practice-two/src/assets/css/pages/home/hero.css]: @hungnguyenhuu I have removed it.

File: practice-two/src/assets/css/pages/home/hero.css (hunk 1 above)
Comment (LeVHoangduc) [reply practice-two/src/assets/css/pages/home/hero.css]: @hungnguyenhuu  I have removed it.

File: practice-two/src/assets/css/pages/home/hero.css [hunk 3]
```diff
@@ -0,0 +1,113 @@
+.hero__mobile.container {
```
Comment (hungnguyenhuu) [inline practice-two/src/assets/css/pages/home/hero.css:1]: Why we have this one? Why don't we style on container? Hard to custom in the future

Comment (LeVHoangduc) [reply practice-two/src/assets/css/pages/home/hero.css:1]: I will update it with another way

File: practice-two/src/assets/css/pages/home/hero.css [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+    display: none;
+  }
+}
+
+@media (min-width: 1200px) {
+  .hero__banner {
+    display: none;
+  }
+
+  .hero__mobile {
```
Comment (hungnguyenhuu) [inline practice-two/src/assets/css/pages/home/hero.css:64]: naming

Comment (LeVHoangduc) [reply practice-two/src/assets/css/pages/home/hero.css]: @hungnguyenhuu  i have update it. Please help me to check it again.

**Comment by hungnguyenhuu** [comment]
conflict

**Pull request merged**